main/lsp/test/lsp_preprocessor_test.cc

```diff
@@ -240,7 +226,7 @@ TEST(LSPPreprocessor, IgnoresWatchmanUpdatesFromOpenFiles) { // NOLINT
 
     string fileContents = "# typed: true\n1+1";
     opts.fs->writeFile("foo.rb", "");
-    preprocessor.preprocessAndEnqueue(state, makeOpen("foo.rb", 1, fileContents), mtx);
+    preprocessor.preprocessAndEnqueue(state, makeOpen("foo.rb", fileContents, 1), mtx);
```

Comment:
The two redundant `makeOpen` methods used different argument orderings. I decided that the trailing version is better.